src/coreclr/vm/reflectioninvocation.cpp

```diff
@@ -1746,7 +1748,9 @@ extern "C" void QCALLTYPE ReflectionSerialization_GetCreateUninitializedObjectIn
 
     TypeHandle type = pType.AsTypeHandle();
 
-    RuntimeTypeHandle::ValidateTypeAbleToBeInstantiated(type, true /* fForGetUninitializedInstance */);
+    // Ref structs can't be boxed (allocated as an uninitialized object).
+    bool allowRefLike = false;
+    ValidateTypeAbleToBeInstantiated(type, allowRefLike, true /* fForGetUninitializedInstance */);
```

Comment:
```suggestion
    ValidateTypeAbleToBeInstantiated(type, false /* allowRefLike */, true /* fForGetUninitializedInstance */);
```
Nit: Use same style for both args

Reply:
(multiple places)